packages/adminrouter/extra/src/README.md

```diff
@@ -197,56 +216,57 @@ http {
   Great example for this is `auth.check_acl_or_exit()` which internally, among
   many other things, calls argument-less `auth.check_jwt_or_exit()`.
 
 In order to address these issues, a couple of patterns were selected:
  * modules which have flavour-specific function arguments export argument-less
-   functions for the NGINX configuration. They translate the original call into
-   a call with correct arguments. This approach requires splitting the module into
-   `ee.lua|open.lua|common.lua` parts which is described in next bullet point.
-   Depending on the flavour, either `ee.lua` or `open.lua` is imported and correct
-   argument-less function is used. This approach also enables to share some of
-   the NGINX `location` blocks between flavours - `location` code is the same,
-   even though the Lua code used by the block differs.
- * some libraries/functions are structured in a way that extracts parts common for
-   both flavours and `ee/open` parts that are included only in EE and Open repos
-   respectively. An example of this approach is auth library which is splitted
-   into three parts:
+     functions for the NGINX configuration. They translate the original call
+   into a call with correct arguments. This approach requires splitting the
+   module into `ee.lua|open.lua|common.lua` parts which is described in next
+   bullet point.  Depending on the flavour, either `ee.lua` or `open.lua` is
+   imported and correct argument-less function is used. This approach also
+   enables to share some of the NGINX `location` blocks between flavours -
+   `location` code is the same, even though the Lua code used by the block
+   differs.
+ * some libraries/functions are structured in a way that extracts parts common
+   for both flavours and `ee/open` parts that are included only in EE and Open
+   repos respectively. An example of this approach is auth library which is
+   splitted into three parts:
     * `lib/auth/common.lua` - present only in Open repository, with functions
       shared by both EE and Open code.
     * `lib/auth/ee.lua` - present only in EE repository, with functions specific
       to EE that use boilerplate from lib/auth/common.lua.
     * `lib/auth/open.lua` - as above but for Open repository.
-   `init_by_lua` OpenResty call in `includes/http/(open|ee)/common.conf` imports
-   `auth.open` or `auth.ee` modules as auth respectively.  The module is registered in
-   the global namespace so all other Lua code uses it. This approach also allows
-   for some degree of code separation enforcement as Open lua libs are removed
-   during EE repository apply.
+   `init_by_lua` OpenResty call in `includes/http/(open|ee)/common.conf`
+   imports `auth.open` or `auth.ee` modules as auth respectively.  The module
+   is registered in the global namespace so all other Lua code uses it. This
+   approach also allows for some degree of code separation enforcement as Open
+   lua libs are removed during EE repository apply.
```

Comment:
"during" should be followed by an accurate description of a process. Please find a more accurate description of a process than "EE repository apply".